### ISSUE TITLE
Output of Two .pb files why the outputs are different , How to make them same.

### ISSUE THREAD

**Issue by sonu107** [tensorflow/tensorflow#19550]
>>> gf.ParseFromString(open('OptimizedGraph.pb','rb').read())5713824
>>> [n.name + '=>' +  n.op for n in gf.node if n.op in ( 'Softmax','Placeholder')]
[u'inputTensor=>Placeholder', u'dropout_keep_prob=>Placeholder']
>>> gf.ParseFromString(open('test.pb','rb').read())4543637
>>> [n.name + '=>' +  n.op for n in gf.node if n.op in ( 'Softmax','Placeholder')]
[u'inputTensor=>Placeholder', u'dropout_keep_prob=>Placeholder', u'output/softmax=>Softmax']
